### PR TITLE
CHAT-562 Hide Call button on mobile

### PR DESCRIPTION
**Pull request by LamiaBouhajja**

### CODE REVIEW

File: application/src/main/java/org/exoplatform/chat/portlet/chat/assets/responsive-767px-max.less [hunk 1]
```diff
@@ -526,6 +526,7 @@
             right: -2px;
         &.hide{display: none!important}
     }
+    #chat-video-button{display: none}
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/assets/responsive-767px-max.less]: It means the button will still be visible on tablets, is it what we want ?

Comment (plamarque) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/assets/responsive-767px-max.less]: No.

[quote=Patrice Lamarque]pas de bouton call ni sur les smartphones ni sur les tablettes [/quote]

Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/assets/responsive-767px-max.less]: So the PR must be updated :smile:

Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/assets/responsive-767px-max.less]: @LamiaBouhajja I am not sure to get your point. With this change, the button will still be visible on tablets.

**Pull request merged**